pymemcache/client/base.py

```diff
@@ -293,7 +293,8 @@ def close(self):
             finally:
                 self.sock = None
 
-    def set(self, key, value, expire=0, noreply=None):
+    def set(self, key, value, expire=0, noreply=None, flag=None,
+            encoding='ascii'):
```

Comment:
It's a best practice to add new keyword arguments to the end of the signature in case someone has made an assumption about the positional order.

Also, I think this should be spelled `flags` to match the memcached protocol. And should this default to `0` instead of `None`?

Reply:
Please ignore the first part of my comment. I mistakenly thought that `encoding` already existed here.